### DISCUSSION TITLE
Copilot suggestions interfere with Resharper tooltop (IntelliSense)

### DISCUSSION THREAD

**Discussion by TuTAH1** [community/community#124529] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Description

When writing the code with Resharper enabled, Copilot suggestions are not displayed when you stop writing the code. But if I start writing, stop and immediately hit ESC (so Resharper's IntelliSense won't appear, I can see Copilot's suggestions. Also, sometimes if Copilot's suggestion appears, the Resharper IntelliSense doesn't appear at all.

### Video
https://skrinshoter.ru/vPnx8wC5126?a

In the video:

1. Copilot suggestion appeared → IntelliSense tooltip no longer appears until I accept Copilot's suggestion.
2. (0:14) I accept and erased Copilot's suggestion. The IntelliSense tooltip appears now, but Copilot's suggestion is not. Pressed esc (hide tooltip) – still no Copilot's suggestions
3. (0:32) I start writing and immediately pressed esc when I stop to cancel autocomplete suggestions tooltip → now there's Copilot's suggestions.

**Comment by ayim** [comment, marked as answer]
Hello from the Visual Studio team - the feedback and repro was very helpful.
Would love comments on where our IntelliSense doesn't match Resharper's.

We'll be tracking and updating this issue here:
https://developercommunity.visualstudio.com/t/GitHub-Copilot-Completions-in-Visual-Stu/10688821

--
Aaron Yim
Product Manager for Visual Studio
aayim@microsoft.com
